src/port.js

```diff
@@ -31,6 +34,28 @@ export default class Port {
 			console.log(msg);
 		}
 	}
+	getService(serviceType, version) {
+		if(!this.isConnected) {
+			throw new Error('Cannot getService() before connect() has completed');
+		}
+		let me = this;
+		return new Promise((resolve, reject) => {
+			me.requestRaw(`service:${serviceType}:${version}`)
```

Comment:
`methodNames` is passed in.  `serviceType` and `version` are only used in a string, so I think you could pass in the string instead of the individual values.

`Q`'s resolve method can take a promise.  If `Promise` works the same way, then you could pass the result of `.then` to resolve.

With all these, I think you could end up with `resolve(me.requestRaw(...).then(helper(serviceVersionPrefix))`.

But I'm not worried about this, just seemed like an opportunity.